packages/beacon-node/test/e2e/network/gossipsub.test.ts

```diff
@@ -19,7 +19,6 @@ describe("gossipsub / worker", function () {
 
 function runTests(this: Mocha.Suite, {useWorker}: {useWorker: boolean}): void {
   if (this.timeout() < 20 * 1000) this.timeout(150 * 1000);
-  this.retries(2); // This test fail sometimes, with a 5% rate.
```

Comment:
Why can this be removed now? Was this just related to libp2p issue?

Reply:
Because tests are performing more stable now, so we don't need to have retries here.